.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+composer.lock
+vendor/
+
```

Comment:
We usually don't have .gitignore files in any of the XP modules. I use https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore